### PR TITLE
Sandbox responsivenesss

### PR DESCRIPTION
**Pull request by langolf**
Continue from https://github.com/badoo/styleguide/pull/80. Bad merge.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.requireConfig": true
```
Comment (Winner95) [inline .vscode/settings.json:2]: if you would like to add this file to repo - could you please add it in a separate  PR with description - what it does and why we need it?

File: yarn.lock [hunk 2]
```diff
@@ -2,52 +2,46 @@
 # yarn lockfile v1
```
Comment (Winner95) [inline yarn.lock:3]: @langolf - let's not run yarn upgrade in a feature PRs please - let's do it in a separate PR

**Review by Winner95 — CHANGES_REQUESTED** [review]

File: src/components/page/page-styles.js [hunk 3]
```diff
@@ -48,4 +49,19 @@ const Pagestyles = useDefaultGlobalStyles
...
     `;
 
+/**
+ * @type {typeof deviceSizes}
+ * @return {string}
+ * Media query helper for "styled-components"
+ */
+export const respondTo = Object.keys(deviceSizes).reduce((accumulator, label) => {
+    /** @type {function(...args)} */
+    accumulator[label] = (...args) => css`
```
Comment (Winner95) [inline src/components/page/page-styles.js:59]: names of parameters - make it relevant to functionality

File: src/components/page/page-styles.js [hunk 4]
```diff
@@ -48,4 +49,19 @@ const Pagestyles = useDefaultGlobalStyles
...
+        @media (min-width: ${deviceSizes[label]}px) {
+            ${css(...args)};
```
Comment (Winner95) [inline src/components/page/page-styles.js:61]: instead of args via spread - let's put one parameter that we can pass

let's cover it in tests as well:

if we pass working / non working code - what expect

File: src/visual-helpers/styleguide-cell/styleguide-cell.jsx [hunk 5]
```diff
@@ -47,22 +62,22 @@ function StyleguideCell(props) {
                 </Text>
             ) : null}
 
-            <View
+            <StyledView
```
Comment (Winner95) [inline src/visual-helpers/styleguide-cell/styleguide-cell.jsx:65]: let's have it in separate repo as it will be published.

**Review by Winner95 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (langolf) [reply .vscode/settings.json:2]: Removed

**Review by Winner95 — APPROVED** [review]
